deploy/terraform/variables.tf

```diff
@@ -61,10 +76,15 @@ variable "mayastor_cpus" {
   description = "number of CPUs to use"
   default     = 2
 }
+variable "mayastor_cpu_mask" {
+  type        = string
+  description = "CPU mask"
```

Comment:
this should read core-list, core-mask is defined with -m...

mask: -m 0x3
or 
list: -l 0,2

Same thing the latter is perhaps what easier to deal with reason about.

Reply:
dohh good catch thanks.

Reply:
I've also added https://github.com/openebs/mayastor-control-plane/pull/107/commits/93acc1dce40ae7d98803b5e26434d7d80625389c which is probably fine for testing and speeds up installs quite a bit.